### ISSUE TITLE
Do you have any idea for estimating the uncertainty of USLE by MCMC in R?

### ISSUE THREAD

**Issue by Renjie-soil** [jbousquin/USLE#1]
As title

**Comment by jbousquin** [comment]
Yes that was the idea here - the script was used to estimate some of the uncertainty around precipitation. Each of the parameters have their own uncertainty. One that I thought could use improvement was sediment delivery ratio at the end, informing it with NOAA's QNSPECT or similar. It's been a long time so there are probably better ways to code it now.
